<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_novo_orcamento</class>
 <widget class="QMainWindow" name="tela_novo_orcamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Orçamento</string>
  </property>
  <widget class="QWidget" name="novo_orcamento">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget_novo_orcamento">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_dados_cliente">
     <attribute name="title">
      <string>Dados do Cliente</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_buscar_cliente">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Localizar Cliente</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_data">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>573</x>
        <y>28</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Data</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>21/05/2020</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_data">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Data:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_endereco_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>82</y>
        <width>190</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Endereço da Obra ou do Cliente</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_telefone_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>32</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Telefone para contato</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_endereco_cliente">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Endereço:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_numero_cliente">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>62</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cidade_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>82</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Cidade da Obra</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_nome_cliente">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome do Cliente:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bairro_cliente">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>62</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bairro:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_vendedor">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>12</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vendedor Responsável:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_editar_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Editar dados do Cliente</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_bairro_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>82</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Bairro da Obra</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_vendedores">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>32</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Vendedor Responsável</string>
      </property>
      <property name="placeholderText">
       <string>Selecione o vendedor</string>
      </property>
      <item>
       <property name="text">
        <string>Selecione o vendedor</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>82</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Numero da Obra</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nome_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>32</y>
        <width>190</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Nome do Cliente</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_cidade_cliente">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>62</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cidade:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_salvar_cliente">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Salvar dados do Cliente</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_telefone_cliente">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>12</y>
        <width>50</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_observacoes">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Observações sobre o Orçamento</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_observacoes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Observações:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_novo_cliente">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Cadastrar novo Cliente</string>
      </property>
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>971</width>
      <height>241</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>160</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>linha 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>linha 2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descrição</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Grade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quant.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor Unit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor Total</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Branco</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>Sem grade</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Preto</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>Com grade</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_incluir_produto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir Produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_produto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Produto:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_lista_produtos">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_excluir_produto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Salvar dados do Cliente</string>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancelar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Cancelar Cadastro de Cliente</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nome_cliente</tabstop>
  <tabstop>lineEdit_telefone_cliente</tabstop>
  <tabstop>comboBox_vendedores</tabstop>
  <tabstop>lineEdit_endereco_cliente</tabstop>
  <tabstop>lineEdit_numero_cliente</tabstop>
  <tabstop>lineEdit_bairro_cliente</tabstop>
  <tabstop>lineEdit_cidade_cliente</tabstop>
  <tabstop>textEdit_observacoes</tabstop>
  <tabstop>pushButton_salvar_cliente</tabstop>
  <tabstop>pushButton_editar_cliente</tabstop>
  <tabstop>pushButton_buscar_cliente</tabstop>
  <tabstop>pushButton_incluir_produto</tabstop>
  <tabstop>lineEdit_data</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
